DependencyInjection/Compiler/WellKnownSchemaFilterPass.php

```diff
@@ -42,7 +42,10 @@ public function process(ContainerBuilder $container)
 
                 case LegacyConnection::class:
                 case Connection::class:
-                    $blacklist[] = $definition->getArguments()[0]['table_name'] ?? 'messenger_messages';
+                    // add the blacklist *unless* the new schema subscriber service exists
+                    if (!$container->getDefinition('doctrine.orm.messenger.doctrine_schema_subscriber')) {
+                        $blacklist[] = $definition->getArguments()[0]['table_name'] ?? 'messenger_messages';
```

Comment:
The idea is that we would transition away from this. The new correct solution would be that the subscriber always adds the table to the schema for you - so it should never be in the schema_filter.

Reply:
Is this needed for the PR? If not, I'd suggest leaving this as is - it's unrelated and still works.